webpack/InsightsCloudSync/Components/InsightsSettings/InsightsSettings.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import InsightsSyncSwitcher from '../InsightsSyncSwitcher';
+import './insightsSettings.scss';
+
+class InsightsSettings extends Component {
+  componentWillMount() {
+    const { getInsightsSyncSettings } = this.props;
+    getInsightsSyncSettings();
```

Comment:
should we pass all of the settings we need as `props` [when mounting the component](https://github.com/theforeman/foreman_rh_cloud/blob/master/app/views/foreman_rh_cloud/react/insights_cloud.html.erb#L5 )

Reply:
This component is not mounted directly. It is mounted from the `InsightsCloudSync` page-component.